model/src/main/kotlin/AdvisorDetails.kt

```diff
@@ -26,7 +26,13 @@ data class AdvisorDetails(
     /**
      * The name of the used advisor.
      */
-    val name: String
+    val name: String,
+
+    /**
+     * The capabilities of the used advisor. This property indicates, which kind of findings are retrieved by the
+     * advisor.
+     */
+    val capabilities: Set<AdvisorCapability> = emptySet()
```

Comment:
The default was introduced, because otherwise old result files cannot be read any more. IMHO, this would be a high price to pay for this new feature.

Reply:
While I agree that each advisor should specify a capability, I think a default is needed to prevent that older result files cannot be loaded any more.